GEOSldas_App/GEOSldas_HISTens.rc

```diff
@@ -0,0 +1,1083 @@
+
+#
+#  Sample GEOSldas HISTORY.rc file for LADAS (atm ensemble)
+#
+#  - This sample HISTORY.rc is for the GEOSldas instance that is weakly coupled with the
+#       atmospheric ensemble component of the Hybrid-4DEnVar ADAS (ADASens).
+#
+#  - The sample file was generated with the utility script
+#       "GEOSldas/src/Components/GEOSldas_GridComp/GEOSldas_App/util/config/generate_catchincr_hist.py".
+#
+#  - The sample file triggers output of the GEOSldas "catch_progn_incr" collection in
+#       ensemble space, which is needed by ADASens.
+#
+#  - The IDs of the ensemble members and their total number in GEOSldas must match
+#       those of ADASens.
+#
+#  - The "catch_progn_incr" output is in tile space, which must be the same for
+#       GEOSldas and ADASens.
+#
+#
+##################################################################################
+
+EXPID:  GEOSldas_expid
+
+COLLECTIONS:
```

Comment:
See comment above -- add "lnd" and "lndfcstana" collections (ens avg)

Reply:
Just to clarify, for "lnd" and "lndfcstana", I'm only thinking of the **ensemble average** collections.  Not "lnd" for each ens member.